codeforamerica/vita-min
# Nj 63 property tax deduction credit

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/63

https://github.com/newjersey/affordability-pm/issues/64

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Add calculations for whether property tax credit or deduction is more beneficial (see logic in ticket)
- Update lines 41, 42, 43, 56 based on calculation

## How to test?
- Use many_w2s or two_w2s to simulate different income levels
- Use MFS to test MFS specific logic, use tenant screen to check if same home
  
## Screenshots (for visual changes)

### When taking property tax deduction:
![image](https://github.com/user-attachments/assets/358638f3-41cf-4387-9722-0a562b0e632f)
![image](https://github.com/user-attachments/assets/bbaf4382-8eef-4a12-af46-091cb80650e9)
![image](https://github.com/user-attachments/assets/085e3c2a-2448-4d7b-a9c5-877cd9dcac8b)

### When taking property tax credit:
![image](https://github.com/user-attachments/assets/333a3b71-a91d-4886-a209-6b18e680ae54)
![image](https://github.com/user-attachments/assets/3472d262-d064-4f9d-a72d-4e3adca48f43)
![image](https://github.com/user-attachments/assets/48ef1c52-25b4-436a-83dd-ce0f0d45bb35)
